betty/project/extension/maps/assets/templates/maps/view-padding.json.j2

```diff
@@ -0,0 +1 @@
+[50, 50, 50, 50]
```

Comment:
Should this become a single file for multiple configuration options, e.g. marker anchors, and cluster distances as well?